Functions.Templates/Templates/HttpTrigger-Python/__init__.py

```diff
@@ -1,20 +1,22 @@
 import logging
 
+# Add any dependencies you might need here

```

Comment:
Dependencies need to be defined in requirements.txt

Reply:
You're right. I'll update this to be more explicit